enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -1179,6 +1179,10 @@ bool ActivityAnalyzer::isConstantValue(TypeResults const &TR, Value *Val) {
   }
 
   if (auto CI = dyn_cast<CallInst>(Val)) {
+    if (CI->hasFnAttr("enzyme_sample")) {
```

Comment:
In the future, we may mark only specific calls as activity sources. Aka whether we need that derivative (as requested).

-> or why don't we just mark as enzyme_active?

Reply:
done